packages/react-sdk/src/components/ImportWhiteboardDialog/ImportWhiteboardDialog.tsx

```diff
@@ -172,11 +252,120 @@ export function ImportWhiteboardDialog({
         <Button
           onClick={handleOnImport}
           variant="contained"
-          disabled={importedWhiteboard?.isError}
+          disabled={error || loading}
         >
           {t('boardBar.importWhiteboardDialog.success.import', 'Import')}
         </Button>
       </DialogActions>
     </Dialog>
   );
 }
+
+function readNWB(file: File): Promise<{
+  name: string;
+  isError: boolean;
+  data?: WhiteboardDocumentExport;
+}> {
+  const reader = new FileReader();
+
+  const logger = getLogger('SettingsMenu');
```

Comment:
This is the import component. Why do we get the logger for the settings menu here?

Reply:
I think this is a merge weirdness that happened when the import dialog was reworked. Will fix/adjust